ANSWERS.md

```diff
@@ -0,0 +1,12 @@
+1. In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+    A. Describe() breaks the test suite into components making it easier to read.  It() is used inside describe() and 
+        actually runs the individual tests.
+
+2. What is the point of `Test Driven Development`? What do you think about this approach?
```

Comment:
Yup spot on 👍 and honestly I don't think anyone does TDD unless they are getting paid to 🤷‍♂️